packages/utils/src/lib/git.integration.test.ts

```diff
@@ -21,12 +30,19 @@ describe('git utils', () => {
 
     await git.add('README.md');
     await git.commit('Create README');
+
+    await git.checkout(['-b', 'feature-branch']);
```

Comment:
Does this mean the first test starts in `feature-branch` but all subsequent tests start in `master` due to the `afterEach`? That is not intuitive, please make it consistent.

Reply:
No it just created the branch

Reply:
done